src/tvshows.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+const instance = axios.create({
+  baseURL: "https://api.themoviedb.org/3",
+  timeout: 1000,
+  headers: { Authorization: "Bearer " + import.meta.env.VITE_API_BEARER_TOKEN },
+});
+
+export async function getTvshows(timeWindow) {
```

Comment:
No need for the `async` here since there is no `away`. Here is a brief explanation on asynchronous JavaScript: https://www.w3schools.com/js/js_async.asp